README.md

```diff
@@ -36,10 +36,11 @@ A planned Polkadot collator for the parachain.
 
 4. Run the collator:
 
-	`cargo run --release -p cumulus-test-parachain-collator -- --base-path cumulus_collator_path --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/PEER_ID --bootnodes /ip4/127.0.0.1/tcp/50666/p2p/PEER_ID`
+	`cargo run --release -p cumulus-test-parachain-collator -- --base-path cumulus_collator_path -- --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/PEER_ID --bootnodes /ip4/127.0.0.1/tcp/50666/p2p/PEER_ID`
```

Comment:
```suggestion
	`cargo run --release -p cumulus-test-parachain-collator -- --base-path cumulus_collator_path -- --bootnodes=/ip4/127.0.0.1/tcp/30333/p2p/PEER_ID_ALICE --bootnodes=/ip4/127.0.0.1/tcp/50666/p2p/PEER_ID_BOB`
```

Two changes here:

- Add an `=` in the flag, which is necessary for it to actually work
- use distinct `PEER_ID_${NAME}` per instructions below